pom.xml

```diff
@@ -252,6 +252,14 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+        	<activation>
+			<jdk>[9,)</jdk>
+            </activation>
+            <properties>
+            	<maven.compiler.release>17</maven.compiler.release>
+           	</properties>
+	    </profile>
```

Comment:
This fails the compilation if you don't have Java 17 installed. Why is that necessary?

Reply:
it fails if you have something JDK>= 9 and it is not 17.
if you compile it with JDK8 it runs as usual.
my intention in the PR is to make Oak work with JDK 17 and that why I put 17 in line 260.

if you have JDK 11 you may change the line from 17 to 11 and then it will work with JDK 11.
Or if you wanna just run with 11 and compile 8, then we can change the activation value from [9, to [17,
then this effect will take place just when we detect system with JDK>= 17

Reply:
So why is it necessary? If you compile on JDK17, then it will compile for Java 17 by default, right?

Reply:
As I understand it, if this is not present even if we JDK=17 installed 
since the maven source-target is set to 8, it will still compile with 8.
In my changes it will run and compile with 17

Reply:
Yes. This is indeed true. I suggest that for now, when compiling for a specific JVM version, simply change JAVA_HOME to the appropriate value.